[CI] MixedClusterClientYamlTestSuiteIT test {p0=aggregations/matrix_stats_multi_value_field/Unmapped} failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/zt53t7yrrhguu/tests/:qa:mixed-cluster:v8.7.2%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=aggregations%2Fmatrix_stats_multi_value_field%2FUnmapped%7D

**Reproduction line:**
```
./gradlew ':qa:mixed-cluster:v8.7.2#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=aggregations/matrix_stats_multi_value_field/Unmapped}" -Dtests.seed=96C5874C54523485 -Dtests.bwc=true -Dtests.locale=en-ZA -Dtests.timezone=Asia/Thimphu -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
No. I ran it 3 times locally from the `e3bd7d2f6ff6c924d68517fdeefae3129d0c6a1b` commit on main and it passed every time.

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT&tests.test=test%20%7Bp0%3Daggregations/matrix_stats_multi_value_field/Unmapped%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [aggregations/matrix_stats_multi_value_field:35]: expected [2xx] status code but api [index] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"mapper [vals] cannot be changed from type [long] to [float]","stack_trace":"org.elasticsearch.ElasticsearchException$1: mapper [vals] cannot be changed from type [long] to [float]\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:668)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:596)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:175)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:123)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:102)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:55)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.client.internal.node.NodeClient$SafelyWrappedActionListener.onFailure(NodeClient.java:170)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:218)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction.lambda$unwrappingSingleItemBulkResponse$0(TransportBulkAction.java:190)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:167)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$RunBeforeActionListener.onResponse(ActionListener.java:440)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$7.onResponse(ActionListener.java:546)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation$1.finishHim(TransportBulkAction.java:658)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation$1.onResponse(TransportBulkAction.java:639)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation$1.onResponse(TransportBulkAction.java:628)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.client.internal.node.NodeClient$SafelyWrappedActionListener.onResponse(NodeClient.java:160)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:209)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:203)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase.finishOnSuccess(TransportReplicationAction.java:1049)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase$1.handleResponse(TransportReplicationAction.java:961)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase$1.handleResponse(TransportReplicationAction.java:952)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1367)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1466)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1437)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:41)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.ChannelActionListener.lambda$onResponse$0(ChannelActionListener.java:38)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener.run(ActionListener.java:567)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.ChannelActionListener.onResponse(ChannelActionListener.java:38)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.ChannelActionListener.onResponse(ChannelActionListener.java:20)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$RunBeforeActionListener.onResponse(ActionListener.java:440)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$7.onResponse(ActionListener.java:546)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.lambda$runWithPrimaryShardReference$3(TransportReplicationAction.java:494)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:167)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$MappedActionListener.onResponse(ActionListener.java:132)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation.finish(ReplicationOperation.java:454)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation.decPendingAndFinishIfNeeded(ReplicationOperation.java:440)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation$2.lambda$onResponse$0(ReplicationOperation.java:226)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation.updateCheckPoints(ReplicationOperation.java:341)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation$2.onResponse(ReplicationOperation.java:226)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation$2.onResponse(ReplicationOperation.java:222)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.RetryableAction$RetryingListener.onResponse(RetryableAction.java:144)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:43)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1367)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:400)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.handleResponse(InboundHandler.java:349)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.messageReceived(InboundHandler.java:144)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.inboundMessage(InboundHandler.java:97)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:829)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundPipeline.forwardFragments(InboundPipeline.java:150)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundPipeline.doHandleBytes(InboundPipeline.java:121)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundPipeline.handleBytes(InboundPipeline.java:86)\n\tat org.elasticsearch.transport.netty4@8.7.2-SNAPSHOT/org.elasticsearch.transport.netty4.Netty4MessageInboundHandler.channelRead(Netty4MessageInboundHandler.java:63)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\n\tat io.netty.handler@4.1.86.Final/io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:280)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\n\tat io.netty.codec@4.1.86.Final/io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:689)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:652)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)\n\tat io.netty.common@4.1.86.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)\n\tat io.netty.common@4.1.86.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalArgumentException: mapper [vals] cannot be changed from type [long] to [float]\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.checkIncomingMergeType(FieldMapper.java:372)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:352)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:55)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.ObjectMapper$Builder.buildMappers(ObjectMapper.java:154)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.RootObjectMapper$Builder.build(RootObjectMapper.java:113)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.createDynamicUpdate(DocumentParser.java:233)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:85)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:78)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:1021)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:969)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperationOnPrimary(IndexShard.java:913)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:350)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction$2.doRun(TransportShardBulkAction.java:216)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:282)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:136)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:73)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportWriteAction$1.doRun(TransportWriteAction.java:212)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:958)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\t... 1 more\n"}],"type":"illegal_argument_exception","reason":"mapper [vals] cannot be changed from type [long] to [float]","stack_trace":"java.lang.IllegalArgumentException: mapper [vals] cannot be changed from type [long] to [float]\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.checkIncomingMergeType(FieldMapper.java:372)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:352)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:55)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.ObjectMapper$Builder.buildMappers(ObjectMapper.java:154)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.RootObjectMapper$Builder.build(RootObjectMapper.java:113)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.createDynamicUpdate(DocumentParser.java:233)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:85)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:78)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:1021)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:969)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperationOnPrimary(IndexShard.java:913)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:350)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction$2.doRun(TransportShardBulkAction.java:216)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:282)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:136)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:73)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportWriteAction$1.doRun(TransportWriteAction.java:212)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:958)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\n"},"status":400}]

  at __randomizedtesting.SeedInfo.seed([96C5874C54523485:1E91B896FAAE597D]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:572)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:515)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

  Caused by: java.lang.AssertionError: expected [2xx] status code but api [index] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"mapper [vals] cannot be changed from type [long] to [float]","stack_trace":"org.elasticsearch.ElasticsearchException$1: mapper [vals] cannot be changed from type [long] to [float]\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:668)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:596)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:175)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:123)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:102)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:55)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.client.internal.node.NodeClient$SafelyWrappedActionListener.onFailure(NodeClient.java:170)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:218)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction.lambda$unwrappingSingleItemBulkResponse$0(TransportBulkAction.java:190)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:167)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$RunBeforeActionListener.onResponse(ActionListener.java:440)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$7.onResponse(ActionListener.java:546)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation$1.finishHim(TransportBulkAction.java:658)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation$1.onResponse(TransportBulkAction.java:639)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportBulkAction$BulkOperation$1.onResponse(TransportBulkAction.java:628)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.client.internal.node.NodeClient$SafelyWrappedActionListener.onResponse(NodeClient.java:160)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:209)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onResponse(TaskManager.java:203)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase.finishOnSuccess(TransportReplicationAction.java:1049)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase$1.handleResponse(TransportReplicationAction.java:961)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase$1.handleResponse(TransportReplicationAction.java:952)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1367)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1466)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1437)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:41)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.ChannelActionListener.lambda$onResponse$0(ChannelActionListener.java:38)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener.run(ActionListener.java:567)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.ChannelActionListener.onResponse(ChannelActionListener.java:38)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.ChannelActionListener.onResponse(ChannelActionListener.java:20)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$RunBeforeActionListener.onResponse(ActionListener.java:440)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$7.onResponse(ActionListener.java:546)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.lambda$runWithPrimaryShardReference$3(TransportReplicationAction.java:494)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:167)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListener$MappedActionListener.onResponse(ActionListener.java:132)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation.finish(ReplicationOperation.java:454)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation.decPendingAndFinishIfNeeded(ReplicationOperation.java:440)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation$2.lambda$onResponse$0(ReplicationOperation.java:226)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation.updateCheckPoints(ReplicationOperation.java:341)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation$2.onResponse(ReplicationOperation.java:226)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.ReplicationOperation$2.onResponse(ReplicationOperation.java:222)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.RetryableAction$RetryingListener.onResponse(RetryableAction.java:144)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:43)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1367)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:400)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.handleResponse(InboundHandler.java:349)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.messageReceived(InboundHandler.java:144)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundHandler.inboundMessage(InboundHandler.java:97)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:829)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundPipeline.forwardFragments(InboundPipeline.java:150)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundPipeline.doHandleBytes(InboundPipeline.java:121)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.transport.InboundPipeline.handleBytes(InboundPipeline.java:86)\n\tat org.elasticsearch.transport.netty4@8.7.2-SNAPSHOT/org.elasticsearch.transport.netty4.Netty4MessageInboundHandler.channelRead(Netty4MessageInboundHandler.java:63)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\n\tat io.netty.handler@4.1.86.Final/io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:280)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\n\tat io.netty.codec@4.1.86.Final/io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:689)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:652)\n\tat io.netty.transport@4.1.86.Final/io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)\n\tat io.netty.common@4.1.86.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)\n\tat io.netty.common@4.1.86.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalArgumentException: mapper [vals] cannot be changed from type [long] to [float]\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.checkIncomingMergeType(FieldMapper.java:372)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:352)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:55)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.ObjectMapper$Builder.buildMappers(ObjectMapper.java:154)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.RootObjectMapper$Builder.build(RootObjectMapper.java:113)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.createDynamicUpdate(DocumentParser.java:233)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:85)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:78)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:1021)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:969)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperationOnPrimary(IndexShard.java:913)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:350)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction$2.doRun(TransportShardBulkAction.java:216)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:282)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:136)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:73)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportWriteAction$1.doRun(TransportWriteAction.java:212)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:958)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\t... 1 more\n"}],"type":"illegal_argument_exception","reason":"mapper [vals] cannot be changed from type [long] to [float]","stack_trace":"java.lang.IllegalArgumentException: mapper [vals] cannot be changed from type [long] to [float]\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.checkIncomingMergeType(FieldMapper.java:372)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:352)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.FieldMapper.merge(FieldMapper.java:55)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.ObjectMapper$Builder.buildMappers(ObjectMapper.java:154)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.RootObjectMapper$Builder.build(RootObjectMapper.java:113)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.createDynamicUpdate(DocumentParser.java:233)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:85)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:78)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:1021)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:969)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.index.shard.IndexShard.applyIndexOperationOnPrimary(IndexShard.java:913)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:350)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction$2.doRun(TransportShardBulkAction.java:216)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:282)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:136)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.bulk.TransportShardBulkAction.dispatchedShardOperationOnPrimary(TransportShardBulkAction.java:73)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.action.support.replication.TransportWriteAction$1.doRun(TransportWriteAction.java:212)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:958)\n\tat org.elasticsearch.server@8.7.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\n"},"status":400}]

    at org.junit.Assert.fail(Assert.java:88)
    at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:373)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:552)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:515)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke(Method.java:578)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
    at java.lang.Thread.run(Thread.java:1623)

```

--- a/modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/matrix_stats_multi_value_field.yml
+++ b/modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/matrix_stats_multi_value_field.yml
@@ -20,7 +20,7 @@ setup:
               "val3":
                 "type": "double"
               "vals":
-                "type": "double"
+                "type": "float"
 
   - do:
       indices.create:

--- a/modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/matrix_stats_multi_value_field.yml
+++ b/modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/matrix_stats_multi_value_field.yml
@@ -19,6 +19,8 @@ setup:
                 "type": "double"
               "val3":
                 "type": "double"
+              "vals":
+                "type": "double"
 
   - do:
       indices.create:

